cmd/sys-mgmt-executor/main.go

```diff
@@ -86,13 +78,17 @@ func main() {
 		err := ExecuteDockerCommands(service, operation)
 
 		if err != nil {
-			agent.LoggingClient.Error(fmt.Sprintf("error performing  %s on service %s: %v", operation, service, err.Error()))
+			LoggingClient.Error(fmt.Sprintf("error performing  %s on service %s: %v", operation, service, err.Error()))
 		} else {
-			agent.LoggingClient.Info(fmt.Sprintf("success performing %s on service %s", operation, service))
+			LoggingClient.Info(fmt.Sprintf("success performing %s on service %s", operation, service))
 		}
 	}
 }
 
+func setLoggingTarget() string {
+	return LoggingTarget
```

Comment:
since this is just a constant is there any possibility that this will become configurable at some point? If not, you should just use the `LoggingTarget` constant in the call to `logger.NewClient` instead

Reply:
Good point: Done ✅ (That was a carry-over from my exploring (and experimenting with), earlier on, how to make the initialization machinery configurable (and subsequently deciding against doing so)).